Directory.Build.targets

```diff
@@ -1,10 +1,18 @@
 <Project>
-
+  <!-- Set code coverage properties that reference properties not available in Directory.Build.props -->
+  <PropertyGroup Condition="'$(Coverage)' == 'true'">
+    <CoverletOutput>$(TargetDir)coverage\$(MSBuildProjectName).coverage</CoverletOutput>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Coverage)' == 'true'">
+    <!-- https://github.com/tonerdo/coverlet/issues/363 -->
+    <DeterministicSourcePaths>false</DeterministicSourcePaths>
+  </PropertyGroup>
+  
   <Import Project="$(WpfArcadeSdkTargets)"
           Condition="Exists('$(WpfArcadeSdkProps)') And Exists('$(WpfArcadeSdkTargets)')"/>
 
   <Import Sdk="Microsoft.DotNet.Arcade.Wpf.Sdk"
           Project="Sdk.targets"
           Condition="!Exists('$(WpfArcadeSdkProps)') Or !Exists('$(WpfArcadeSdkTargets)')"/>
-
+  <Import Project="eng/Testing.targets"/>
```

Comment:
Also, how about importing this along with other props and targets file in Sdk.targets ? Seems like a better place for being imported ?